Nodejs/Product/TestAdapter/AssemblyResolver.cs

```diff
@@ -11,64 +11,56 @@ internal sealed class AssemblyResolver : IDisposable
     {
         public AssemblyResolver()
         {
-            AppDomain.CurrentDomain.AssemblyResolve += OnAssemblyResolve;
+            // Use the setup API to find the VS install Dir, then build paths to the Private and Public Assemblies folders
+            var installPath = GetVSInstallDir();
+            var ideFolder = Path.Combine(installPath, "Common7", "IDE");
+            this.probePaths = new[] {
```

Comment:
>probePaths [](start = 17, length = 10)

This worries me.  VS doesn't already have a probe path?

Reply:
Yes, but this runs in the 'testhost' which doesn't do the right thing

Reply:
I suppose there's no API for retrieving VS's prove path?

Reply:
Nope, not that I could find.